workshop/README.md

```diff
@@ -0,0 +1,15 @@
+## Dataset description
+
+The dataset is used in [VS Code workshop](https://github.com/iterative/VSCode-DVC-Workshop) 
+for predicting satellite kinematic orbit and run several experiments.
+
+The dataset contains information about RSOs - artificial objects that are in 
+orbit around earth used to predict orbit trajectories. 
+
+Each row represents an orbit observation of a satellite. There are 600 
+satellites that have multiple observations.
+
+### Acknowledgments
+
+Dataset published in [Kaggle](https://www.kaggle.com/datasets/idawoodjee/predict-the-positions-and-speeds-of-600-satellites) and has been collected by
```

Comment:
same here, please format

there are plugins for VS Code that can do wrap automatically - e.g. https://marketplace.visualstudio.com/items?itemName=stkb.rewrap